packages/react-native/ios/DashXData.swift

```diff
@@ -10,6 +10,17 @@ struct TrackRequest: Encodable {
     let data: JSONValue?
 }
 
+struct SubscribeRequest: Encodable {
+    let value, kind: String
+    let anonymousUid, uid: String?
+}
+
+struct ContentRequest: Encodable {
+    let contentType, returnType: String
```

Comment:
Shouldn't returnType be optional? And if so, shouldn't [line 178](https://github.com/dashxhq/dashx-js/pull/32/files#diff-9adf5fd95421614c544a9e5e122b68d0R178) of `packages/react-native/ios/DashXClient.swift` read `returnType: optionsDictionary?["returnType"] as? String` instead of `as! String`?

Reply:
`returnType` will always be there because native's `content` will be called after `.all` or `.one`. Although there was an issue with the typecasting in `content` method which is fixed now.

Reply:
I just realised, it won't be present when it's called without `ContentBuilderOptions`